subsystem/elevator.py

```diff
@@ -0,0 +1,52 @@
+import rev
+from robotpy_toolkit_7407 import Subsystem
+from robotpy_toolkit_7407.motors.rev_motors import SparkMax, SparkMaxConfig
+from robotpy_toolkit_7407.sensors.limit_switches import MagneticLimitSwitch
+
+import config
+import constants
+from units.SI import meters
+
+# TODO: Change config once robot is built
+ELEVATOR_CONFIG = SparkMaxConfig(
+    1.5, 0, 0.004, 0.00017, (-1, 1), idle_mode=rev.CANSparkMax.IdleMode.kBrake
+)
+
+
+class Elevator(Subsystem):
+
+    motor_extend: SparkMax = SparkMax(
+        config.elevator_can_id, config=ELEVATOR_CONFIG, inverted=True
+    )
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.elevator_bottom_sensor = None
+
+    def init(self) -> None:
+        self.motor_extend.init()
+        self.elevator_bottom_sensor = MagneticLimitSwitch(config.magnetic_limit_switch_port)
+
+    def stop(self) -> None:
+        """
+        Stops the elevator
+        """
+        self.motor_extend.set_raw_output(0)
```

Comment:
when it comes to stopping the elevator, you need to use a set target position, otherwise, the elevator will drop due to it being outside of the closed loop. I would recommend grabbing the current position of the motor, then setting the target value to that. That way, even though the elevator has stopped, it is not moving up or down based on external factors